flybywiresim/a32nx
# chore: replacing XML tooltips, missing id for seatbelt switch

Fixes #[issue_no]

## Summary of Changes


Replacing strings with .locPak cockpit tooltips where possible for better localisation support, readded missing <Instrument Id> for seatbelt signs. 

Added (Tooltip -> EN):
* ``TT:COCKPIT.TOOLTIPS.STROBE_LIGHT_OFF`` : "Turn ON strobe lights"
* ``TT:COCKPIT.TOOLTIPS.STROBE_LIGHT_ON`` : "Turn OFF strobe lights"
* ``TT:COCKPIT.TOOLTIPS.LIGHTS_READING_DECREASE`` :  "Decrease reading light brightness"
* ``TT:COCKPIT.TOOLTIPS.LIGHTS_READING_INCREASE``  :  "Increase reading light brightness"

There will be no visible change to the end user if they were already using English as their language. 

``<PART_ID>Seatbelt_Switch</PART_ID>`` - Affects checklists (Eye button to see relevant switch). Seems to have been broken at some point, tied it into this PR. 

## Screenshots (if necessary)



* New Strobe light tooltips (No visual change from before)
![FlightSimulator_bp1EYmD9ao](https://user-images.githubusercontent.com/15316958/120102567-39852800-c17e-11eb-9246-0c185cc386c5.png)

* New Reading lights tooltips (No visual change from before)
![FlightSimulator_NIh6KKabzp](https://user-images.githubusercontent.com/15316958/120102561-31c58380-c17e-11eb-9cf1-c752d530a232.png)
![FlightSimulator_LzW5t0zCjo](https://user-images.githubusercontent.com/15316958/120102564-3427dd80-c17e-11eb-99fb-748ffacc3894.png)

* Seatbelt switch in checklist (Fix)
![image](https://user-images.githubusercontent.com/15316958/120102556-25412b00-c17e-11eb-9b20-493eea3fd36a.png)


## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Check that See seatbelt switch "eye" button prompt shows up in relevant checklist entries 
2. Check that updated tooltips are working and not broken.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
